TESTING-INSTRUCTIONS.md

```diff
@@ -2,6 +2,107 @@
 
 ## Unreleased
 
+### Payment gateway suggestions feature
+
+1. Note that the `payment-gateway-suggestion` feature should be turned on in your current build
```

Comment:
How would someone from GlobalStep check this exactly? Is there a specific zip file we have that has it enabled (that we can provide)?

Reply:
Good question.  Initially I made these test instructions with Isotope in mind, but I do think it will go to GlobalStep eventually so I'll remove this.

This will be turned on in all environments as of https://github.com/woocommerce/woocommerce-admin/pull/7224